apps/laboratory/playwright.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig<ModalFixture>({
   expect: {
     timeout: getValue(60, 15) * 1000
   },
-  timeout: 60 * 1000,
+  timeout: 90 * 1000,
```

Comment:
I increased the test timeout to not cause conflicts with other tests, since i added a test which takes 30+ seconds.